### ISSUE TITLE
Dependency mentioned in README.md can not be found

### ISSUE THREAD

**Issue by flyrev** [lightbody/browsermob-proxy#211]
Both
```XML
    <dependency>
      <groupId>net.lightbody.bmp</groupId>
      <artifactId>browsermob-core</artifactId>
      <version>2.1.0-beta-1-SNAPSHOT</version>
      <scope>test</scope>
    </dependency>
```
and
```XML
    <dependency>
      <groupId>net.lightbody.bmp</groupId>
      <artifactId>browsermob-core-littleproxy</artifactId>
      <version>2.1.0-beta-1-SNAPSHOT</version>
      <scope>test</scope>
    </dependency>
```

both end with "Could not resolve dependencies".

After a quick Google search, I could only find http://mvnrepository.com/artifact/net.lightbody.bmp/browsermob-proxy/2.0.0 ?

**Comment by jekh** [comment]
The -SNAPSHOT ending means that it is referring to the latest build in master. To use a snapshot version, you'll need to build yourself from the command line (see the bottom of the readme file). I'll be releasing a non-snapshot version to Central "soon", as soon as a few key defects are fixed.

**Comment by flyrev** [comment]
Ah, okay. May I suggest putting a POM example in there that would work out of the box? :)))

**Comment by jekh** [comment]
The README.md file in master is in sync with the code in master. At the top of the readme there is a link to the readme for the most recently released version, which is 2.0.

**Comment by flyrev** [comment]
Thanks!
